### PR TITLE
Plot examples during the Sprint

### PR DESCRIPTION
**Pull request by jscarbor**
Still working on examples from the PyDataLA Sprint with Ravin

### CODE REVIEW

**Comment by jscarbor** [comment]
OK. I verified today my draft is what I intend to PR. I tool out [WIP] in the name. Please proceed. I'm happy to make corrections. Thank you @canyon289 for the newbie support!

**Comment by canyon289** [comment]
Thanks @jscarbor. I'll check this tomorrow and pending no issues I'll merge.

I don't see any, just good practice to check. Thanks in advance!

**Comment by OriolAbril** [comment]
LGTM, thanks!

File: arviz/plots/distplot.py [hunk 1]
```diff
@@ -90,6 +90,48 @@ def plot_dist(
...
     ax : matplotlib axes
+
+    Examples
+    --------
+    Plot an integer distribution
+
+    .. plot::
+        :context: close-figs
+
+        >>> import matplotlib.pyplot as plt
```
Comment (OriolAbril) [inline arviz/plots/distplot.py]: Being very picky, this line is actually not used, otherwise it is completely spotless

Comment (canyon289) [reply arviz/plots/distplot.py]: Ah good catch. @jscarbor could you delete just this one line? After that this will be ready to merge.

Comment (jscarbor) [reply arviz/plots/distplot.py]: Good catch indeed @OriolAbril . Done. And I confirmed that plot_dist.py where I copied those import from does in fact utilize the import.

**Comment by OriolAbril** [comment]
Thanks!

**Comment by jscarbor** [comment]
A pleasure meeting and working with you all!

**Pull request merged**